src/app/views/initial-setup/initial-setup-name/initial-setup-name.component.html

```diff
@@ -2,13 +2,11 @@
   <app-close-icon></app-close-icon>
   <div class="initial-setup-name__logo">
     <img alt="Green Traveller Logo" src="assets\icons\android-chrome-512x512.png" height="128" width="128">
+    <h3 class="initial-setup-name__headline">Green Traveller</h3>
```

Comment:
just a thought, only a suggestion: maybe add the emojis with the name?

Reply:
tried, is too much stuff then